### ISSUE TITLE
deeplabv3+ eval.py result

### ISSUE THREAD

**Issue by hyemin002** [tensorflow/models#5190]
### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

**Even though the loss value falls. the same mIOU value(0.03491) always comes out**

 python train.py --logtostderr 
--training_number_of_steps=30000 
--train_split='train' 
--model_variant='xception_65' 
--atrous_rates=6 
--atrous_rates=12 
--atrous_rates=18 
--output_stride=16 
--decoder_output_stride=4 
--train_crop_size=513 
--train_crop_size=513 
--train_batch_size=5 
--dataset='pascal_voc_seg' 
--train_logdir="/storage/models-master/research/deeplab/datasets/pascal_voc_seg/exp/train_on_train_set/train/DGX_0824_1" 
--dataset_dir="/storage/models-master/research/deeplab/datasets/pascal_voc_seg/tfrecord" 
--tf_initial_checkpoint="/storage/models-master/research/deeplab/datasets/pascal_voc_seg/init_models/deeplabv3_pascal_train_aug/model.ckpt" --fine_tune_batch_norm=false

python eval.py --logtostderr 
--eval_split='val' 
--model_variant='xception_65' 
--atrous_rates=6 
--atrous_rates=12 
--atrous_rates=18 
--output_stride=16 
--deocder_output_stride=4 
--eval_crop_size=513 
--eval_crop_size=513 
--checkpoint_dir="/storage/models-master/research/deeplab/datasets/pascal_voc_seg/exp/train_on_train_set/train/DGX_0824_1" 
--eval_logdir="/storage/models-master/research/deeplab/datasets/pascal_voc_seg/exp/train_on_train_set/val/DGX_0824_1" 
--dataset_dir="/storage/models-master/research/deeplab/datasets/pascal_voc_seg/tfrecord" 
--max_number_of_evaluations=1

Total_loss is about 0.16~0.18
mIOU 0.03491

-----------------------------------------------------------------------------------------------------------

python train.py --logtostderr 
--training_number_of_steps=10 
--train_split='train' 
--model_variant='xception_65' 
--atrous_rates=6 
--atrous_rates=12 
--atrous_rates=18 
--output_stride=16 
--decoder_output_stride=4 
--train_crop_size=513 
--train_crop_size=513 
--train_batch_size=5 
--dataset='pascal_voc_seg' 
--train_logdir="/storage/SEG_jhm2/models-master/research/deeplab/datasets/pascal_voc_seg/exp/train_on_train_set/train/Titan_0827_test" 
--dataset_dir="/storage/SEG_jhm2/models-master/research/deeplab/datasets/pascal_voc_seg/tfrecord" --tf_initial_checkpoint="/storage/SEG_jhm2/models-master/research/deeplab/datasets/pascal_voc_seg/init_models/deeplabv3_pascal_train_aug/model.ckpt" --fine_tune_batch_norm=false

python eval.py --logtostderr 
--eval_split='val' 
--model_variant='xception_65' 
--atrous_rates=6 
--atrous_rates=12 
--atrous_rates=18 
--output_stride=16 
--deocder_output_stride=4 
--eval_crop_size=513 
--eval_crop_size=513 
--checkpoint_dir="/storage/SEG_jhm2/models-master/research/deeplab/datasets/pascal_voc_seg/exp/train_on_train_set/train/Titan_0827_test" 
--eval_logdir="/storage/SEG_jhm2/models-master/research/deeplab/datasets/pascal_voc_seg/exp/train_on_train_set/val/Titan_0827_test" 
--dataset_dir="/storage/SEG_jhm2/models-master/research/deeplab/datasets/pascal_voc_seg/tfrecord" --max_number_of_evaluations=1

mIOU 0.03491

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by Z-Ianthe** [comment]
I have the same problem. Have you solved it？Could you give me some suggestions

**Comment by yangzh11** [comment]
I have the same problem. Have you solved it？Could you give me some suggestions
